docs/hub/model-release-checklist.md

```diff
@@ -4,6 +4,16 @@ The Hugging Face Hub is the go-to platform for sharing machine learning models.
 
 ## ⏳ Preparing Your Model for Release
 
+### Uploading models
+
+When uploading models to the hub, it's recommended to follow a set best practices:
+
+- push weights to separate model repositories. Example: prefer uploading individual quantizations/precisions in a standalone repo like [this](https://huggingface.co/jameslahm/yolov10n) over all types/versions in one like [this](https://huggingface.co/kadirnar/Yolov10/tree/main).
```

Comment:
It would look better if the list was capitalized

Reply:
Will submit a follow-up PR

Reply:
Nvm just saw https://github.com/huggingface/hub-docs/pull/1660, I assume that resolves it